### PR TITLE
[SDPA-4532] remove redundant include query

### PR DESCRIPTION
**Pull request by tim-yao**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-4532

### Changed

1.  removed redundant include query. Previously if we need related entity `c` from entity `a`( like `a.b.c` ), we have to use `include=a&a.b&a.b.c`. Now it can be just `include=a.b.c`. 

### Screenshots

Before: landing page API request include query (3781 characters)
```
field_node_site,field_node_site.field_site_main_menu,field_node_site.field_site_og_image,field_node_site.field_site_og_image.field_media_image,field_featured_image,field_featured_image.field_media_image,field_tags,field_topic,field_related_links,field_whats_next,field_graphical_image,field_graphical_image.field_media_image,field_bottom_graphical_image,field_bottom_graphical_image.field_media_image,field_landing_page_hero_logo,field_landing_page_hero_logo.field_media_image,field_landing_page_hero_image,field_landing_page_hero_image.field_media_image,field_landing_page_hero_banner,field_landing_page_c_primary,field_landing_page_c_primary.field_block_image,field_landing_page_c_primary.field_block_image.field_media_image,field_landing_page_c_secondary,field_landing_page_c_secondary.field_block_image,field_landing_page_c_secondary.field_block_image.field_media_image,field_landing_page_c_secondary.field_block_embedded_video,field_landing_page_key_journeys,field_landing_page_contact,field_landing_page_contact.field_paragraph_phones,field_landing_page_contact.field_paragraph_social_media,field_landing_page_header,field_landing_page_component,field_landing_page_component.field_paragraph_media,field_landing_page_component.field_paragraph_media.field_media_image,field_landing_page_component.field_paragraph_topic,field_landing_page_component.field_timeline,field_landing_page_component.field_timeline.field_paragraph_media,field_landing_page_component.field_timeline.field_paragraph_media.field_media_image,field_landing_page_component.field_paragraph_accordion,field_landing_page_component.field_paragraph_keydates,field_landing_page_component.field_paragraph_reference,field_landing_page_component.field_paragraph_reference.field_topic,field_landing_page_component.field_paragraph_reference.field_featured_image,field_landing_page_component.field_paragraph_reference.field_featured_image.field_media_image,field_landing_page_component.field_paragraph_media_gallery,field_landing_page_component.field_paragraph_media_gallery.field_gallery_media,field_landing_page_component.field_paragraph_media_gallery.field_gallery_media.field_media_image,field_landing_page_component.field_paragraph_items,field_landing_page_component.field_paragraph_items.field_paragraph_reference,field_landing_page_component.field_paragraph_items.field_paragraph_reference.field_event_details,field_landing_page_component.field_paragraph_items.field_paragraph_reference.field_topic,field_landing_page_component.field_paragraph_items.field_paragraph_reference.field_featured_image,field_landing_page_component.field_paragraph_items.field_paragraph_reference.field_featured_image.field_media_image,field_landing_page_component.field_paragraph_items.field_paragraph_keydates,field_landing_page_component.field_paragraph_items.field_paragraph_media,field_landing_page_component.field_paragraph_items.field_paragraph_media.field_media_image,field_landing_page_component.field_complex_image_media,field_landing_page_component.field_complex_image_media.field_media_image,field_landing_page_component.field_paragraph_reference.field_event_details,field_landing_page_component.field_paragraph_cta_card_event,field_landing_page_component.field_paragraph_cta_card_event.field_paragraph_media,field_landing_page_component.field_paragraph_cta_card_event.field_paragraph_media.field_media_image,field_landing_page_component.field_paragraph_news_reference,field_landing_page_component.field_paragraph_news_reference.field_featured_image,field_landing_page_component.field_paragraph_news_reference.field_featured_image.field_media_image,field_landing_page_component.field_paragraph_news_reference.field_topic,field_landing_page_component.field_paragraph_topic,field_landing_page_component.field_paragraph_webform
```
Now: include query (2268 characters)
```
field_node_site.field_site_main_menu,field_node_site.field_site_og_image.field_media_image,field_featured_image.field_media_image,field_tags,field_topic,field_related_links,field_whats_next,field_graphical_image.field_media_image,field_bottom_graphical_image.field_media_image,field_landing_page_hero_logo.field_media_image,field_landing_page_hero_image.field_media_image,field_landing_page_hero_banner,field_landing_page_c_primary.field_block_image.field_media_image,field_landing_page_c_secondary.field_block_image.field_media_image,field_landing_page_c_secondary.field_block_embedded_video,field_landing_page_key_journeys,field_landing_page_contact.field_paragraph_phones,field_landing_page_contact.field_paragraph_social_media,field_landing_page_header,field_landing_page_component.field_paragraph_media.field_media_image,field_landing_page_component.field_paragraph_topic,field_landing_page_component.field_timeline.field_paragraph_media.field_media_image,field_landing_page_component.field_paragraph_accordion,field_landing_page_component.field_paragraph_keydates,field_landing_page_component.field_paragraph_reference.field_topic,field_landing_page_component.field_paragraph_reference.field_featured_image.field_media_image,field_landing_page_component.field_paragraph_media_gallery.field_gallery_media.field_media_image,field_landing_page_component.field_paragraph_items.field_paragraph_reference.field_event_details,field_landing_page_component.field_paragraph_items.field_paragraph_reference.field_topic,field_landing_page_component.field_paragraph_items.field_paragraph_reference.field_featured_image.field_media_image,field_landing_page_component.field_paragraph_items.field_paragraph_keydates,field_landing_page_component.field_paragraph_items.field_paragraph_media.field_media_image,field_landing_page_component.field_complex_image_media.field_media_image,field_landing_page_component.field_paragraph_reference.field_event_details,field_landing_page_component.field_paragraph_cta_card_event.field_paragraph_media.field_media_image,field_landing_page_component.field_paragraph_news_reference.field_featured_image.field_media_image,field_landing_page_component.field_paragraph_news_reference.field_topic,field_landing_page_component.field_paragraph_webform
```

### CODE REVIEW

File: packages/ripple-nuxt-tide/modules/news/tide.config.js [hunk 1]
```diff
@@ -6,15 +6,10 @@ module.exports = {
     ],
 
     landingPage: [
-      'field_landing_page_component.field_paragraph_news_reference',
-      'field_landing_page_component.field_paragraph_news_reference.field_featured_image',
       'field_landing_page_component.field_paragraph_news_reference.field_featured_image.field_media_image',
-      'field_landing_page_component.field_paragraph_news_reference.field_topic',
-      'field_landing_page_component.field_paragraph_topic'
```
Comment (tim-yao) [inline packages/ripple-nuxt-tide/modules/news/tide.config.js:13]: 'field_landing_page_component.field_paragraph_topic' is already in landing page module, no need to add again.

File: packages/ripple-nuxt-tide/modules/publication/tide.config.js [hunk 2]
```diff
@@ -1,53 +1,32 @@
...
-      'field_landing_page_component.field_timeline',
-      'field_landing_page_component.field_timeline.field_paragraph_media',
       'field_landing_page_component.field_timeline.field_paragraph_media.field_media_image',
       'field_landing_page_component.field_paragraph_accordion',
       'field_landing_page_component.field_paragraph_keydates',
-      'field_landing_page_component.field_paragraph_reference',
       'field_landing_page_component.field_paragraph_reference.field_topic',
-      'field_landing_page_component.field_paragraph_reference.field_featured_image',
       'field_landing_page_component.field_paragraph_reference.field_featured_image.field_media_image',
-      'field_landing_page_component.field_paragraph_webform',
```
Comment (tim-yao) [inline packages/ripple-nuxt-tide/modules/publication/tide.config.js:37]: 'field_landing_page_component.field_paragraph_webform' will be added by webform module.

**Comment by tim-yao** [comment]
Hey @anthony-malkoun It seems the large url is not an issue now on our server. But if we let it keep growing(adding more components to the landing page continually), may will be an issue in the future? By reducing the query to nearly half in this PR, I feel less risk now.

BTW, the Drupal field name will impact on the include query length, especially when a parent field has a long name. For example, `field_landing_page_component`.

**Review by anthony-malkoun — APPROVED** [review]
I'll have to take your word this works @tim-yao :). I get what we're aiming for so code review is fine and hopefully it doesn't raise in issues in regression.

**Pull request merged**